PARTNERS.md

```diff
@@ -0,0 +1,24 @@
+<!--
+  -- Copyright (c) 2019, Arm Limited, All Rights Reserved
+  -- SPDX-License-Identifier: Apache-2.0
+  --
+  -- Licensed under the Apache License, Version 2.0 (the "License"); you may
+  -- not use this file except in compliance with the License.
+  -- You may obtain a copy of the License at
+  --
+  -- http://www.apache.org/licenses/LICENSE-2.0
+  --
+  -- Unless required by applicable law or agreed to in writing, software
+  -- distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  -- WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  -- See the License for the specific language governing permissions and
+  -- limitations under the License.
+--->
+
+# Contributing Organizations
```

Comment:
Would this be complementary to what's in `CONTRIBUTORS.md`?

Reply:
Yes. See below. PARTNERS need not necessarily be contributors. They may be integrators or adopters.

Reply:
Makes sense!